_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2021-12-31
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Wow - what a year! Here are some notable achievements, failures, and learnings from Artsy Engineering in 2021...
+
+<!-- more -->
+
+## Success!
+
+- Leveraged our investment in React Native to launch our Android App: Props to David, Mounir, Pavlos, Joanna, and the Collector Experience team
+- Moved X% of Force apps to V2 and Y% of Force traffic: Props to Chris, Damon, (...)
+- We killed X repos and retired N services
+- The awards for fastest PR from start date goes to: A! And even better, Ole did his first PR to Artsy 3 months before he joined.
+- We started and released N episodes of the Artsy Engineering Radio
+- We onboarded 23 engineers in Berlin - Props to Adam B, Christian and a whole team of willing mentors
+- Took down our poor CWV url numbers by 876786% (not verified)
+- Design system and Palette docs
```

Comment:
It'd be worth expanding on this a bit in the sense that Palette has been out for a few years, but it was always a WIP until now. With latest docs and v2 work we're pretty much feature complete.